src/rail/estimation/algos/deepdisc.py

```diff
@@ -80,10 +80,15 @@ def train(config, all_metadata, train_head=True):
 
 
     saveHook = return_savehook(run_name)
-    lossHook = return_evallosshook(val_per, model, eval_loader)
     schedulerHook = return_schedulerhook(optimizer)
-    #hookList = [lossHook, schedulerHook, saveHook]
-    hookList = [schedulerHook, saveHook]
+
+    if training_percent >= 1.0:
+        # don't do lossHook
+        hookList = [schedulerHook, saveHook]
+        print(f"The lossHook has been omitted, as the training percent is {training_percent}. To include it, set the training percent to a value between 0 and 1.")
```

Comment:
Could this be rephrased to "The validation loss has been omitted ...."?  It's a bit more intuitive than lossHook

Reply:
Sure thing! Included in the most recent commit